### ISSUE TITLE
`./wpt run` runs all tests by default, no clear way to run just affected tests

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt#13564]
When making test changes in a wpt checkout, `./wpt run`'s default behavior of running all tests is not very useful. I'd go further and say that to a human operator, running all tests is *very* seldom the right thing. Two ideas to improve upon this:
- `--all` to really run all tests
- `--affected` to only run affected tests

The default behavior could be to do nothing, or run the affected tests. Making this change would require coordinating with all uses of `./wpt run` in CI.

@jgraham @jugglinmike

**Comment by gsnedders** [comment]
Note this of course runs into the same issue as with everything else using "affected tests": what's the baseline branch?

**Comment by jgraham** [comment]
Right, I agree; `--affected` isn't really something that makes sense unless it's `--affected=revish` or something.

As for running nothing rather than running everything, it seems like a matter of taste; I think I sometimes do `wpt run firefox` just to see if things work at all. I also think that if `--totalChunks` and `--thinkChunk` are passed then the harness should certainly run some tests; running an entire chunk from CI in order to reproduce a failure is a reasonable (and often not untenably slow) thing to do.

**Comment by gsnedders** [comment]
> Right, I agree; `--affected` isn't really something that makes sense unless it's `--affected=revish` or something.

I mean, presumably it could do the same as `./wpt tests-affected` by default. I still don't like the idea of expanding the weird heuristics.

**Comment by foolip** [comment]
I don't have very specific preferences about what an `--affected` would do by default, perhaps we can  make no guesses, explicit options, and have a bare `./wpt run` show a short help text. A first-time user of `./wpt run` asked me just today if it's possible to run a single test, so it's not entirely clear to people at first what to do.

I agree that having the use of chunking parameters bypass such a behavior makes sense, and would also make the change less risky.

We could go into detail about how this would work, but first, do you think it's a good idea to change the default at all?
